### PR TITLE
Hiding one popup should not affect other popups on the page

### LINKED ISSUE #32: Calling `.hide()` on one popup actually hides all popups on the page
I added a test for this in 300a8999.

### PR DESCRIPTION
**Pull request by robinhouston**
Fixes kiln/flourish-popup#32 

This PR essentially just reverts commit 33ddc8429716e05e788122012d898aa57c72b0e2. A test for this issue has already been committed to master.

### CODE REVIEW

**Comment by robinhouston** [comment]
Once this is merged, we need to:

* Republish this package to NPM;
* Update Flourish to use the new version.

**Review by timbrock — APPROVED** [review]
LGTM

**Comment by daanlouter** [comment]
I think this would probably bring back the bug that was fixed by the revered commit right? https://github.com/kiln/flourish-popup/commit/33ddc8429716e05e788122012d898aa57c72b0e2

**Comment by robinhouston** [comment]
> I think this would probably bring back the bug that was fixed by the reverted commit right? [33ddc84](https://github.com/kiln/flourish-popup/commit/33ddc8429716e05e788122012d898aa57c72b0e2)

Indeed it would, @daanlouter. But this is a much more serious bug!

Can you say more about the bug that @hughsk was trying to fix? I don’t see an issue for it.

**Comment by daanlouter** [comment]
I can't see it either, maybe it was in one of the templates. I think in some instances the constrainer was visible and had a fixed height, if then the window gets resized, the constrainer still takes up the same space and creates scrollbars. So I think this would be a regression. I do think it probably make sense to hide the constrainer if the popup is also not visible. So maybe there's another fix for this problem?

**Comment by robinhouston** [comment]
@daanlouter and I have just been discussing this. He will add a summary of what we concluded to issue kiln/flourish-templates#1403. But we’re going to go ahead and merge this.

**Pull request merged**
